### PR TITLE
[OSDEV-1411] Django admin: Fix editing of contributor list

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
Fix [OSDEV-1411](https://opensupplyhub.atlassian.net/browse/OSDEV-1411). 

Previously, when a user enters `[]` (or leaves the field empty), Django Admin interpreted it as "empty input" and failed validation because the field is marked as required.

Update `parsing_errors` field:
1. Set `blank=True`: Updated the model field to allow blank input in forms. This tells Django Admin to treat the field as optional.
2. Set `null=True`: This allows the field to have a `NULL` value in the database if explicitly set to null in the request.

[OSDEV-1411]: https://opensupplyhub.atlassian.net/browse/OSDEV-1411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by roman-stolar — APPROVED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -27,7 +29,7 @@ to modify moderation event `status`.
 * *Describe architecture/environment changes here.*
 
 ### Bugfix
-* *Describe bugfix here.*
+* [OSDEV-1411](https://opensupplyhub.atlassian.net/browse/OSDEV-1411) - Django admin: fixed issue when update facility list with empty array `Parsing errors` field.
```
[the remark this answers is not in the document]
Comment (VadimKovalenkoSNF) [reply doc/release/RELEASE-NOTES.md]: I refined this, thank you.

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by roman-stolar — APPROVED** [review]
LGTM

**Pull request merged**